main.py

```diff
@@ -14,6 +14,8 @@
 class CommandArgs():
     provider: str = "openai"
     evaluator: str = "openai"
+    model_name: Optional[str] = "gpt-3.5-turbo-0125"
```

Comment:
`model_name` defined 2 times. Please check line 40 below.

Reply:
Thanks for catching this!